public/app/plugins/panel/timeseries/utils.ts

```diff
@@ -128,12 +128,29 @@ export function prepareGraphableFields(
   }
 
   if (frames.length) {
+    setClassicPaletteIdxs(frames, theme);
     return frames;
   }
 
   return null;
 }
 
+const setClassicPaletteIdxs = (frames: DataFrame[], theme: GrafanaTheme2) => {
+  let seriesIndex = 0;
+
+  frames.forEach((frame) => {
+    frame.fields.forEach((field) => {
+      if (field.type === FieldType.number) { // || FieldType.enum
```

Comment:
```suggestion
      if (field.type === FieldType.number) {
```

Is commented out line needed?

Reply:
it's a reminder to change it once we have enum field types (https://github.com/grafana/grafana/pull/60491). where else to keep these notes?

Reply:
Hm, maybe we should link to that WIP PR in the code itself to be more explicit 🤷 That and / or link this line of code on that WIP PR as part of the changes to be made for that to be "done"

Reply:
I agree with @nmarrs, leaving this comment is fine, but let's add a bit of context

Reply:
already changed